src/pyrad_proc/pyrad/io/io_aux.py

```diff
@@ -343,7 +344,7 @@ def get_fieldname_pyart(datatype):
     return field_name
 
 
-def get_file_list(datadescriptor, starttime, endtime, cfg, scan=None):
+def get_file_list(datadescriptor, starttime, endtime, cfg, scan=None, lteconvention = True):
```

Comment:
Done! path_convention is defined in the second line of the main processing file, then incorporated in cfg and in datacfg